cli/internal/helm/client.go

```diff
@@ -278,6 +279,16 @@ func (c *Client) upgradeRelease(
 		}
 		joinServiceVals["attestationConfig"] = string(attestationConfigJSON)
 
+		// v2.8 updates the disk type of GCP default storage class
+		// This value is not updatable in Kubernetes, but we can use a workaround to update it:
+		// First, we delete the storage class, then we upgrade the chart,
+		// which will recreate the storage class with the new disk type.
+		if conf.GetProvider() == cloudprovider.GCP {
+			if err := c.kubectl.DeleteStorageClass(ctx, "encrypted-rwo"); err != nil {
```

Comment:
Wondering if PVCs created with the old storage class will still trigger volume deletion, once the new storage class with another disk type is deployed.

Daniel already said he would test this again.

Reply:
>will still trigger volume deletion, once the new storage class with another disk type is deployed.

This works without problem, and I already tested this yesterday.
Only thing I am unsure of is if volumes can be properly deleted if there is no storageclass at all

Reply:
Deletion works even if the storage class doesnt exist anymore